pkg/cmd/step/create/step_create_task.go

```diff
@@ -974,6 +978,10 @@ func (o *StepCreateTaskOptions) modifyEnvVars(container *corev1.Container, globa
 			Name:  "PREVIEW_VERSION",
 			Value: "${inputs.params.version}",
 		})
+		envVars = append(envVars, corev1.EnvVar{
+			Name:  "PREVIEW_VERSION_LOWER",
+			Value: strings.ToLower("${inputs.params.version}"),
```

Comment:
This isn’t going to do what you want - it’s just lowercasing the already lowercase `${inputs.params.version}`. I think you want to add an addition parameter at https://github.com/jenkins-x/jx/blob/master/pkg/cmd/step/create/step_create_task.go#L1298 and then have this env var reference that parameter.

Reply:
ok cool - thanks!  will try that